vm/src/frame.rs

```diff
@@ -577,9 +578,10 @@ impl Frame {
             }
             bytecode::Instruction::StoreLocals => {
                 let locals = self.pop_value();
+                let locals = objdict::py_dict_to_attributes(&locals);
```

Comment:
Okay, the problem is that I create here a copy of the locals dictionary. It should be a reference to the same dict.